stdlib.sh

```diff
@@ -687,6 +687,32 @@ layout_node() {
   PATH_add node_modules/.bin
 }
 
+# Usage: layout nodenv <node version number>
+#
+# Example:
+#
+#    layout nodenv 15.2.1
+#
+# Uses use_node and layout_node to add the chosen node version and 
+# "$PWD/node_modules/.bin" to the PATH
+#
+layout_nodenv() {
+  local node_version="${1}"
+  local node_versions
+  local nodenv_version
+  node_versions="$(nodenv root)/versions"
+  nodenv_version=${node_versions}/${node_version}
+  if [[ -e "$nodenv_version" ]]; then
+      # Put the selected node version in the PATH
+      NODE_VERSIONS="${node_versions}" NODE_VERSION_PREFIX="" use_node "${node_version}"
+      # Add $PWD/node_modules/.bin to the PATH
+      layout_node
+  else
+    log_error "nodenv: version '$node_version' not installed"
```

Comment:
What do you think of adding the command needed to install the version as well?

Reply:
Good suggestion.  Added instructions at https://github.com/jamesboehmer/direnv/blob/2151074144f76d226e61adec835e7474de625413/stdlib.sh#L1098